packages/cross_file/tool/run_tests.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Runs `dart test -p chrome` in the root of the cross_file package.
+//
+// Called from the custom-tests CI action.
+//
+// usage: dart run tool/run_tests.dart
+// (needs a `chrome` executable in $PATH, or a tweak to dart_test.yaml)
+import 'dart:io';
+import 'package:path/path.dart' as p;
+
+Future<void> main(List<String> args) async {
+  final Directory workingDir = _findPackageDir('pubspec.yaml', 'cross_file');
+
+  final int status = await _runProcess(
+    'dart',
+    <String>[
+      'test',
+      '-p',
+      'chrome',
+    ],
+    workingDirectory: workingDir.path,
+  );
+
+  exit(status);
+}
+
+// Find the Directory that contains the `marker` file, and has `boundary` in its path.
+Directory _findPackageDir(String marker, String boundary) {
+  Directory workingDir = Directory.current;
```

Comment:
Can't we just use the parent directory of `Platform.script`?

Reply:
Haha yes, I think?

Reply:
I wish `File.currentScript` or similar was a thing. `Platform.script` is a `Uri`, and going from `Uri` to `Directory` is a little bit iffy.